### ISSUE TITLE
Project governance

### ISSUE THREAD

**Issue by apdavison** [python-quantities/python-quantities#171]
This project was started by @ddale and he is still, as far as I know, the only administrator (although a few people, including myself, have write permissions and can merge pull requests/make releases). However in recent years, he has not had time to oversee the development of the project.

I would like to suggest that we define some sort of governance structure, so that more than one person has admin permissions, and that there are defined procedures for passing on maintainership to new people when necessary, while ensuring that the original vision of @ddale is respected.

**Comment by rgerkin** [comment]
I agree.  I rely on this package and would be excited to see active governance.

**Comment by ddale** [comment]
Hello everyone.

Andrew, I appreciate your comments. My career and personal life have evolved in directions that don't afford much time for software development. I think it is wonderful that there are people out there who continue to value the project. I think it would be appropriate to pass on administrator duties, and I support developing some kind of governance structure.

I think it would be great if the original vision of the project were respected, but only if that vision is continues to be justified and stand on its own merits.
